sdks/python/apache_beam/transforms/core.py

```diff
@@ -1975,10 +1990,13 @@ def add_input_types(transform):
       return combined
 
     if self.has_defaults:
-      combine_fn = (
-          self.fn if isinstance(self.fn, CombineFn) else
-          CombineFn.from_callable(self.fn))
-      default_value = combine_fn.apply([], *self.args, **self.kwargs)
+      combine_fn = copy.deepcopy(self.fn) if isinstance(self.fn, CombineFn) \
```

Comment:
nit: [prefer](https://www.python.org/dev/peps/pep-0008/#:~:text=The%20preferred%20way%20of%20wrapping,a%20backslash%20for%20line%20continuation.) using () to line continuation token.